src/app/component/ProductReviewForm/ProductReviewForm.style.scss

```diff
@@ -0,0 +1,168 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+.ProductReviewForm {
+    &-Title {
+        margin-bottom: 0;
+    }
+
+    &-ProductName {
+        font-weight: 700;
+        font-size: 2rem;
+
+        @include mobile {
+            font-size: 1.8rem;
+        }
+    }
+
+    .Form {
+        max-width: 500px;
+
+        @include mobile {
+            max-width: none;
+        }
+
+        input,
+        textarea {
+            width: 100%;
+        }
+    }
+
+    &-ReviewRating {
+        margin-top: 1rem;
+    }
+
+    &-ReviewRatingCode {
+        font-size: 1.05rem;
+        font-weight: 700;
+        border: 0;
+        padding: 0;
+    }
+
+    &-RatingOptionGroup {
+        display: flex;
+
+        &::before {
+            content: '☆☆☆☆☆';
+            font-size: 3rem;
+            color: var(--color-primary-base);
+        }
+    }
+
+    .Field {
+        width: 100%;
+
+        &_type_radio {
+            margin-top: 0;
+            position: absolute;
+
+            label {
+                position: absolute;
+                display: block;
+                border: 0;
+                background-color: transparent;
+                width: auto;
+                height: auto;
+                opacity: 0;
+                font-size: 1rem;
+                font-weight: 400;
+                transition-property: opacity;
+                transition-duration: .15s;
+
+                &::after {
+                    font-size: 3rem;
+                    position: relative;
+                    color: var(--color-primary-base);
+                }
+
+                &:hover {
+                    opacity: 1;
+
+                    &::after {
+                        background-color: transparent;
+                    }
+                }
+            }
+
+            &:nth-of-type(1) {
```

Comment:
Please use `flex-direction: column`, use `:checked ~` to make previous stars appear selected. This requires all labels to be visible.

Reply:
Would this be possible as currently inputs are enclosed in Field wrapper? e.g. 
```
<div class="Field Field_type_radio">
    <input type="radio" name="ratings[1]" id="Quality_1" value="1">
    <label for="Quality_1"></label>
</div>
```

Reply:
No, due Field incapsulation it won't be possible. At least, try not specifying them like: one, two, three, four stars all the time. Don't spend too much.

Reply:
Don't fix this